modules/rubiconAnalyticsAdapter.js

```diff
@@ -163,13 +152,13 @@ function sendMessage(auctionId, bidWonId) {
   let referrer = config.getConfig('pageUrl') || (auctionCache && auctionCache.referrer);
   let message = {
     eventTimeMillis: Date.now(),
-    integration,
-    ruleId,
+    integration: rubiConf.int_type || DEFAULT_INTEGRATION,
+    ruleId: rubiConf.rule_name,
```

Comment:
if never set, then it will be `undefined` here and thus not sent after the stringify occurs!